src/lib/db/access-store.ts

```diff
@@ -160,4 +166,33 @@ export class AccessStore {
             })
             .delete();
     }
+
+    async getRootRolesForAllUsers(): Promise<IRoleAndUser[]> {
+        const rows = await this.db
+            .select('id', 'name', 'user_id')
+            .from(`${T.ROLES} AS r`)
+            .leftJoin(`${T.ROLE_USER} AS ru`, 'r.id', 'ru.role_id')
+            .where('r.type', '=', 'root');
```

Comment:
Are we actually finding users that are root (as in, they have full access?)

Reply:
So "root" is not full access, but rather the roles that exists on the "root resources", according to our [Unleash RBAC spcification](https://docs.google.com/document/d/1T2iuM9qgbgPfNMqXoQKEgdEG--a93lFzMsJHHuT0MHI/edit) 

In hindsight I see that "root" can be misleading. Should we change it?